extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -0,0 +1,510 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { MigrationLocalStorage } from '../models/migrationLocalStorage';
+import * as loc from '../models/strings';
+import { MigrationStatusDialog } from '../dialog/migrationStatusDialog';
+
+interface IActionMetadata {
+	title?: string,
+	description?: string,
+	link?: string,
+	iconPath?: { light: string | vscode.Uri; dark: string | vscode.Uri },
+	command?: string;
+}
+
+const maxWidth = 800;
+
+export class DashboardWidget {
+
+	private _migrationStatusCardsContainer!: azdata.FlexContainer;
+	private _view!: azdata.ModelView;
+	/**
+	 * Creates new instance of dashboard
```

Comment:
This comment isn't really accurate - this doesn't actually create anything with just the constructor call